Assets/Scripts/Saving/TempScript.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TempScript : MonoBehaviour
+{
+    public InventoryItemData itemData;
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Alpha0))
+        {
+            SaveLoadManager.Instance.SaveGameToSaveFile();
+        }
+        if (Input.GetKeyDown(KeyCode.Alpha1))
+        {
+            SaveLoadManager.Instance.LoadGameFromSaveFile();
+        }
+        if (Input.GetKeyDown(KeyCode.Alpha2))
+        {
+            GameObject.FindGameObjectWithTag("Player").GetComponent<InventoryHolder>().InventorySystem.AddToInventory(itemData, 3, out _);
```

Comment:
If the player is the only one with an inventory it might be smart to find object with the `InventoryHolder` component. Alternatively, you can find the player by the attached player component.

Reply:
> If the player is the only one with an inventory it might be smart to find object with the `InventoryHolder` component. Alternatively, you can find the player by the attached player component.

What do you mean by that second sentence?

Reply:
> > If the player is the only one with an inventory it might be smart to find object with the `InventoryHolder` component. Alternatively, you can find the player by the attached player component.
> 
> What do you mean by that second sentence?

`GameObject.FindObjectOfType<PlayerController>();`